tests/ref/absolute_img_stacking_a.html

```diff
@@ -0,0 +1,11 @@
+<style>
+body {
+   margin: 0px;
+}
+
+img {
+   position: absolute;
+   margin: 0px;
+}
+</style>
+<img src="rust.png"/>
```

Comment:
That seems like it kind of tailors our reftests to Servo's bugs. I think I'd prefer a test like `opacity_stacking_context_a.html` that actually takes advantage of CSS 2.1 Appendix E paint order rules to force different rendering if a stacking context is or is not created.